### ISSUE TITLE
Fixed positioning is completely broken in IE6 using ie7.js

### ISSUE THREAD

**Issue by GoogleCodeExporter** [liudonghua123/ie7-js#58]
```
What steps will reproduce the problem?
1. fixed positioning test on the IE7.js website
2. URL: http://ie7-js.googlecode.com/svn/test/fixed.html

What is the expected output? What do you see instead?
Fixed positioning of the 2 divs in the example. NEITHER div is fixed, both 
scroll with the page

What version of the product are you using? On what operating system?
IE 6/win-XP SP2

Please provide any additional information below.
I've attached a screenshot. In this picture, the left (yellow) box has scrolled 
entirely off the
screen and the red box has scrolled to the top (whereas, it should be fixed at 
bottom right)

```

Original issue reported on code.google.com by `hur...@gmail.com` on 20 Mar 2008 at 9:12

Attachments:
- [ie7-fixed.jpg](https://storage.googleapis.com/google-code-attachments/ie7-js/issue-58/comment-0/ie7-fixed.jpg)

**Comment by GoogleCodeExporter** [comment]
```
Are you testing on a full-blow IE6 setup or one of the multiple-IE 
installations? If
the latter, have you installed IE8 beta recently.
```

Original comment by `dean.edw...@gmail.com` on 22 Mar 2008 at 3:11

**Comment by GoogleCodeExporter** [comment]
```
I am testing on a full blown IE6 setup and the only browser installed is IE6.

 I have never installed either IE7 or IE8 on that machine. I tested this on a different machine (running windows 
2000 and IE6) and absolute position works fine on that machine. 

The machine that shows the bug with absolute positioning is running windows 
XP/SP2 (not windows 2000) 
and a slightly later version of IE6 so maybe that has something to do with it.

To recap:
1) Absolute positioning test broken: windows XP SP2, IE6 6.0.2900.2180 xpsp_xp2
2) Absolute positioning test works:   windows 2000,    IE6 6.0.2800.1106



```

Original comment by `hur...@gmail.com` on 23 Mar 2008 at 5:48

**Comment by GoogleCodeExporter** [comment]
```
I can't recreate this problem.
```

Original comment by `dean.edw...@gmail.com` on 14 Feb 2010 at 12:14
- Changed state: **Invalid**
